### DISCUSSION TITLE
Not a bug but help appreciated

### DISCUSSION THREAD

**Discussion by bernique** [SVG-Edit/svgedit#532] [New feature request]
This is not a bug, but a request. I would like to have a shared svg editor on my centos 7 server to have many friends writing a birthday card to my brother.
I have tried to follow the doc to open always the same file in /example/name.svg without prompting for saving the settings, but I cannot succeed. It can be just an url I would share.
Thank you for your help!
Loïc

**SVG-Edit environment (IMPORTANT)**
- File for SVG-Edit:`svg-editor.html
- Version: latest stable
- Protocol: http

**Desktop (please complete the following information):**
 - OS: W10
 - Browser all

**Comment by brettz9** [comment]
By "not prompting for saving the settings", do you mean the prompt about local storage permissions or hiding the save button or ... ?

**Comment by sjernigan** [comment]
It will be fairly easy to start the editor with a default image (storagePrompt=false url=/path/to/your/image) but you will need to do some work on your own to store the image back and make it available for the next user.  Seems like more work than I would go through for my brother's birthday. :-(  Perhaps just use a shared google doc?

**Comment by bernique** [comment]
Sorry for my late reply, I was quite busy and the event I wanted SVG-Edit for is now past.
Yes, I wanted to have no prompt for saving settings locally with access to a single file that could then be edited by different users. The idea was indeed to make a group birthday card. I have not enough knowledge to tweak the scripts.
I remain interested for some future use, but if it is too complex, I can close this thread.
Thanks for your feedback.

**Comment by jfhenon** [comment]
I'm closing this thread. We don't intend to support this multi-user mode but we can always review a PR if needed. 
Thanks
